app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+import app.errors as errors
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise errors.NotVaccinatedError(
+                "NotVaccinatedError should be raised with a message"
+            )
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():

```

Comment:
Use method `get` to avoid `KeyError`
```suggestion
        if visitor["vaccine"].get("expiration_date") < datetime.date.today():

```